src/dvclive/live.py

```diff
@@ -664,7 +664,7 @@ def log_sklearn_plot(
             raise InvalidPlotTypeError(name)
 
         sklearn_kwargs = {
-            k: v for k, v in kwargs.items() if k not in plot_config or k != "normalized"
+            k: v for k, v in kwargs.items() if k not in plot_config or k == "normalized"
```

Comment:
```suggestion
            k: v for k, v in kwargs.items() if k not in plot_config
```

We don't need any check for normalized here. It's already part of `plot_config`, which is all that's needed. We don't want to include it in `sklearn_kwargs` since it's purely to set the dvc plots template to `confusion_normalized`.